docker/ubuntu-14.04/Dockerfile

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 # See below for why the MY_ prefix is used.
-ARG MY_REGISTRY
-FROM ${MY_REGISTRY}/ubuntu:trusty
+ARG MY_REGISTRY=
```

Comment:
Wasn't this already done in 9f27b5a19ef8ddfa49c2d57d437bdd2b88812afa ?

Reply:
This makes it so that not specifying MY_REGISTRY as an argument results in it using whatever the docker daemon defaults to (which doesn't have to be dockerhub).

It does mean that MY_REGISTRY has to be suffixed with "/", but pyrex takes care of that for you.

Reply:
Pyrex was already taking care of ensuring that MY_REGISTRY was set, so this doesn't really seem like it's streamlining the user experience in that sense. What's the motivation to just accept dockerd's default registry then? Maybe we expect that the local administrator has configured the fastest known mirror already?

Reply:
Ya, that was the idea. Hardcoding the default to docker.io isn't equivalent to not specifying a location. Specifically, if you did have a daemon with a different registry the pyrex default of "docker.io" is probably the wrong thing and it would be annoying to *also* have the alternate in the pyrex.ini file.

Being able to leave MY_REGISTRY blank resolves this, as it make it punt to whatever the docker daemons default is.

Reply:
Okay, fair enough.